adlfs/multithread.py

```diff
@@ -69,8 +72,8 @@ class ADLDownloader(object):
     adlfs.transfer.ADLTransferClient
     """
     def __init__(self, adlfs, rpath, lpath, nthreads=None, chunksize=2**28,
-                 blocksize=2**22, client=None, run=True, overwrite=False,
-                 verbose=True):
+                 buffersize=2**25, blocksize=2**22, client=None, run=True,
```

Comment:
is this the "optimal" buffer/block combo based on your tests? I would recommend defaulting to the one that performed the best.